### DISCUSSION TITLE
[blocks]: Multistep form

### DISCUSSION THREAD

**Discussion by od-code** [shadcn-ui/ui#3694] [Blocks Request]
### Description

Multistep form with a progress bar which onboards a user collecting the necessary information.

**Comment by anishshobithps** [comment]
@od-code  there is a [PR for Stepper function](https://github.com/shadcn-ui/ui/pull/318),

Also there are [some great examples](https://github.com/damianricobelli/shadcn-stepper/tree/main/examples) you can find by the author of the PR, it has w.r.t to Form as well.
